packages/backend/src/assets/ai.json

```diff
@@ -86,6 +86,23 @@
       ],
       "backend": "llama-cpp"
     },
+    {
+      "id": "chatbot-nodejs",
+      "description": "This is a Node.js based chat demo application.",
+      "name": "Node.js based ChatBot",
+      "repository": "https://github.com/containers/ai-lab-recipes",
+      "ref": "v1.3.0",
```

Comment:
I don't see the sources at the tag 1.3.0. I suppose it should be `1.3.0.1`

Reply:
Sorry I rename the tag (as I wrongly named it) and the latest is now v1.3.3.1

Reply:
But because of the fix I think it need a new tag

Reply:
as commented below, we need this PR to land https://github.com/containers/ai-lab-recipes/pull/825 and then a new tag and then we can update this PR to point to that tag.